generator/generator.yml

```
@@ -1211,3 +1211,19 @@ modules:
       - source_indexes:
         - bvValue
         lookup: bvDescription
+
+# Generic UPS Mibs from Observioum covering 
+# 1.3.6.1.2.1.33.1
+
+  ups_mib:
+    walk:
+      - 1.3.6.1.2.1.33.1
```

I prefer to be a bit more specific than the whole MIB tree. Also use the text names.

```suggestion
      - "UPS-MIB::upsIdent"
      - "UPS-MIB::upsBattery"
      ...
```

---

No worries! Thanks for the contribution.